lib/Github/Api/Apps.php

```diff
@@ -9,6 +9,15 @@
  */
 class Apps extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    public function configure()
+    {
+        $this->acceptHeaderValue = 'application/vnd.github.machine-man-preview+json';
+
+        return $this;
+    }
```

Comment:
I'm not too sure whether this part is needed?

```suggestion
    public function configure()
    {
        $this->acceptHeaderValue = 'application/vnd.github.machine-man-preview+json';
    }
```

Reply:
TIL Multiline suggestions don't work
My point, not too sure if we need the `return $this;`
Not having anything causes the test's to fail

Reply:
The `return $this` is indeed not needed here, so you can remove it. Which tests fail without the `return $this`?

Reply:
I see why they will fail, just rename this method to `configurePreviewHeader`, make the method final and don't have it return anything. This should work correctly!